zenodo/modules/deposit/indexer.py

```diff
@@ -70,3 +70,8 @@ def indexer_receiver(sender, json=None, record=None, index=None,
     else:
         json['_updated'] = record.updated
     json['_created'] = record.created
+
+    # Compute filecount and total file size
+    files = json.get('_files', [])
+    json['filecount'] = len(files)
+    json['filesize'] = sum([f.get('size', 0) for f in files])
```

Comment:
`filesize` as a field name might give the impression of searching for the individual file size (Example query: `filesize:>1000000`), except of course if one is thinking of a record as a single file/archive. Alternatives: `record_size`, `total_filesize`, `dataset_size`, `archive_size`.